src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button } from "../Button/Button";
+import { Icon } from "../Icon/Icon";
+import React from "react";
+
+type ModalType = "default" | "large" | "forcedAction";
```

Comment:
There is no large modal with forced action in the design specs for either Figma or USWDS. This would be a good question for HCD if they'd like to add this.